src/System.Private.CoreLib/src/System/Utf8String.Manipulation.cs

```diff
@@ -77,11 +77,9 @@ public Utf8String Substring(int startIndex, int length)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public Utf8String Substring(Range range)
-        {
-            (int start, int length) = range.GetOffsetAndLength(Length);
-            return Substring(start, length);
-        }
+        [ComponentModel.EditorBrowsable(ComponentModel.EditorBrowsableState.Never)]
+        public Utf8String Slice(int startIndex, int length) => Substring(startIndex, length);
```

Comment:
Should probably have a comment that we don't intend for developers to call this method directly; it's pattern-matched by the compiler to provide indexer functionality.

Reply:
I'll do that.